pkg/nodelabels/builder.go

```diff
@@ -22,23 +22,25 @@ import (
 )
 
 const (
-	RoleLabelName15        = "kubernetes.io/role"
-	RoleLabelName16        = "kubernetes.io/role"
-	RoleMasterLabelValue15 = "master"
-	RoleNodeLabelValue15   = "node"
+	RoleLabelName15              = "kubernetes.io/role"
+	RoleLabelName16              = "kubernetes.io/role"
+	RoleControlPlaneLabelValue15 = "master"
```

Comment:
This change strikes me as being misleading.

Reply:
You want to keep the master name in this case?

Reply:
Because the value will have to continue to be "master", yes.